dev/io.openliberty.xmlws.3.0.internal.tools.wsgen/bnd.bnd

```diff
@@ -0,0 +1,56 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/cmdline-subdir.props
+bVersion=1.0
+
+Bundle-Name: IBM XML Web Services WsGen Tool
+Bundle-SymbolicName: io.openliberty.xmlWS.3.0.internal.tools.wsgen
+
+Require-Compiler: true
+
+Require-Bundle:\
+ io.openliberty.xmlWS.3.0.internal.tools; version="[1,1.0.100)", \
+ io.openliberty.xmlBinding.3.0.internal.tools; version="[1,1.0.100)", \
+ io.openliberty.jakarta.activation.2.0;version="[1,1.0.100)";location="dev/api/spec/", \
+ io.openliberty.jakarta.jws.3.0;version="[1,1.0.100)";location="dev/api/spec/", \
+ io.openliberty.jakarta.soap.2.0;version="[1,1.0.100)";location="dev/api/spec/", \
+ io.openliberty.jakarta.xmlWS.3.0;version="[1,1.0.100)";location="dev/api/spec/", \
+ io.openliberty.jakarta.xmlBinding.3.0;version="[1,1.0.100)";location="dev/api/spec/", \
+ io.openliberty.com.sun.xml.messaging.saaj; version="[1,1.0.100)", \
+ com.ibm.ws.logging; version="[1,1.0.100)"
+ 
+Parent-Last-Package: jakarta.xml.bind,\
```

Comment:
Is this needed any longer?  I assumed it was because of stuff being in the JDK, but now stuff isn't in the JDK any longer.